deps/Makefile

```diff
@@ -658,6 +658,12 @@ $$(LLVM_SRC_DIR)/$1.patch-applied: $(LLVM_SRC_DIR)/configure | $$(SRCDIR)/$1.pat
 	echo 1 > $$@
 LLVM_PATCH_LIST += $$(LLVM_SRC_DIR)/$1.patch-applied
 endef
+define LLVM_PATCH0
+$$(LLVM_SRC_DIR)/$1.patch-applied: $(LLVM_SRC_DIR)/configure | $$(SRCDIR)/$1.patch
+	cd $$(LLVM_SRC_DIR) && patch -p0 < $$(SRCDIR)/$1.patch
```

Comment:
I wanted to keep the original LLVM patch without modification.

I can, of course, modify it -- but then it might be easier to just append it to one of the existing patches instead of introducing a new one.